src/app/(main)/main/page.tsx

```
@@ -1,73 +1,32 @@
-            handleGraduationProgress={handleGraduationProgress}
-            earnedCredits={data?.summary.earnedCredits ?? 0}
-            requiredCredits={data?.summary.requiredCredits ?? 0}
-          />
-        </>
-      )}
-    </div>
+    <>
+      <AsyncBoundary>
+        <ProfileCard />
+      </AsyncBoundary>
+      <div className="gap-16"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)